upup/pkg/fi/cloud.go

```diff
@@ -27,6 +27,23 @@ type Cloud interface {
 	ProviderID() CloudProviderID
 
 	DNS() (dnsprovider.Interface, error)
+
+	// FindVPCInfo looks up the specified VPC by id, returning info if found, otherwise (nil, nil)
+	FindVPCInfo(id string) (*VPCInfo, error)
+}
+
+type VPCInfo struct {
```

Comment:
Well, generally keeping aws in the AWS package, lest AWS-isms creep in.  I'm sure they have anyway, but having the separation will help us if we go to another cloud.